cli/src/commands/log.rs

```diff
@@ -104,6 +105,11 @@ pub(crate) fn cmd_log(
         Some(value) => value.to_string(),
         None => command.settings().config().get_string("templates.log")?,
     };
+    let use_elided_nodes = command
+        .settings()
+        .config()
+        .get_bool("log.synthetic-elided-nodes")
```

Comment:
need to update `config-schema.json`.

Optionally, the default can be defined in `config/misc.toml` and error out on missing/invalid value.